src/edu/wright/cs/raiderplanner/model/Room.java

```diff
@@ -46,46 +48,92 @@ protected void replace(VersionControlEntity receivedVce) {
 	// public methods
 
 	// getters
+	/**
+	 * Default method to return the building this room resides in.
+	 * @return building.
+	 */
 	public Building getBuilding() {
 		return building;
 	}
 
+	/**
+	 * Default method to return the room number.
+	 * @return roomNumber.
+	 */
 	public String getRoomNumber() {
 		return roomNumber;
 	}
 
+	/**
+	 * Method to output the name and room number of a person.
+	 * If the room does not have a building number, the
+	 * output message displays "name - Unknown Building".
+	 * If the building is not null, the output message displays
+	 * "name ( room ) located in building" where room represents
+	 * the room number and building represents the building object.
+	 * @return String.
```

Comment:
This is not useful, as I can tell the return type by looking at the method signature.